packages/ui/src/proposals/modals/AddNewProposal/AddNewProposalModal.tsx

```diff
@@ -70,6 +71,7 @@ export const AddNewProposalModal = () => {
   const maximumReferralCut = api?.consts.members.referralCutMaximumPercent
   const minCashoutAllowed = api?.consts.content.minimumCashoutAllowedLimit
   const maxCashoutAllowed = api?.consts.content.maximumCashoutAllowedLimit
+  const palletFrozenStatus = useObservable(() => api?.query.projectToken.palletFrozen(), [api?.isConnected])
```

Comment:
Also on this proposal the value of `updatePalletFrozenStatus.frozen` be set to `!palletFrozenStatus` by default. Otherwise when the form appears the value might already be wrong so the next button is disabled but it's not clear why:
![image](https://github.com/Joystream/pioneer/assets/6571453/aa0a4648-f220-4d82-a672-5cac275e8f17)